tests/components/airzone_cloud/test_climate.py

```diff
@@ -356,6 +443,24 @@ async def test_airzone_climate_set_temp(hass: HomeAssistant) -> None:
     state = hass.states.get("climate.group")
     assert state.attributes.get(ATTR_TEMPERATURE) == 20.5
 
+    # Installations
+    with patch(
+        "homeassistant.components.airzone_cloud.AirzoneCloudApi.api_put_installation",
+        return_value=None,
+    ):
+        await hass.services.async_call(
+            CLIMATE_DOMAIN,
+            SERVICE_SET_TEMPERATURE,
+            {
+                ATTR_ENTITY_ID: "climate.house",
+                ATTR_TEMPERATURE: 20.5,
+            },
+            blocking=True,
+        )
+
+    state = hass.states.get("climate.house")
+    assert state.attributes.get(ATTR_TEMPERATURE) == 20.5
```

Comment:
```suggestion
    assert state.attributes[ATTR_TEMPERATURE] == 20.5
```

Reply:
Done in 76940724d61cfd07afc5f515f7d90d91acbec322